src/BioFSharp.Stats/RNASeq.fs

```diff
@@ -0,0 +1,72 @@
+namespace BioFSharp.Stats
+
+open System
+open System.Collections.Generic
+
+
+module RNASeq = 
```

Comment:
At least the public functions and types in this module should have XML documentation.